app/models/report.rb

```diff
@@ -82,7 +83,9 @@ def runtime
   end
 
   def self.import(report, proxy_id = nil)
```

Comment:
Looks like the controller needs refreshing to use this capability, and so it selects the right importer.